tests/commands/got.py

```diff
@@ -35,3 +35,7 @@ def test_cmd_got(self):
         res = gdb.execute("got printf", to_string=True)
         self.assertIn("printf", res)
         self.assertNotIn("strcpy", res)
+
+        res = gdb.execute("got --all printf", to_string=True)
+        self.assertIn("printf", res)
+        self.assertNotIn("strcpy", res)
```

Comment:
What can we test that will show up in `got --all` but won't show up without it?

Reply:
These checks should make sure that they show up UNDER the libc part.

You could get the res, assert `/libc` is in there, split on it, and then make sure `printf` and `strcpy` are on the second part.

Reply:
Split doesn't really work, because it would return two sections: everything before libc and everything after.  AFAIK, the order isn't guaranteed, so the program binary could be in either one.

(Also, printf and strcpy are expected in the program's section, but not in libc's, because those aren't relocatable symbols in libc.)

If you'd like, I can rewrite this to parse the sections...  I'll upload a commit that does that.  If you don't like it, I'll drop that commit and roll back to the current state.